data/json/proficiencies/food.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "type": "proficiency",
+    "id": "prof_asian_food",
+    "name": { "str": "Cooking Asian Food" },
```

Comment:
Maybe it'd be better to name those "Asian Cuisine" and "European Cuisine", respectively? Sounds better to me. And perhaps description in English, too.

Reply:
I considered that too, I'm down with it.